deps/libgit2.mk

```diff
@@ -90,9 +72,14 @@ $(LIBGIT2_SRC_PATH)/libgit2-ssh-loop.patch-applied: $(LIBGIT2_SRC_PATH)/source-e
 		patch -p1 -f < $(SRCDIR)/patches/libgit2-ssh-loop.patch
 	echo 1 > $@
 
-$(build_datarootdir)/julia/cert.pem: $(CERTFILE)
+$(build_datarootdir)/julia/cert.pem:
+	$(eval TMP_CACERT_SHA256:=$(shell mktemp cacert-2017-09-20.XXXX.pem.sha256))
+	echo "435ac8e816f5c10eaaf228d618445811c16a5e842e461cb087642b6265a36856 *cacert-2017-09-20.pem" > $(TMP_CACERT_SHA256)
+	$(JLDOWNLOAD) $(shell pwd)/cacert-2017-09-20.pem https://curl.haxx.se/ca/cacert-2017-09-20.pem
```

Comment:
Is there a rule of thumb that says when we should bump up to newer `cacert` releases?  We might need to add some kind of automated reminder so that we don't forget about it.  What is a reasonable timeframe to bump this up?  Once a month?  Once every six months?

Reply:
Well, according the dates of past cacert.pem revisions ([here](https://curl.haxx.se/docs/caextract.html)), the updates have come out between 2 months to 5 months.